exec/jdbc/src/test/java/org/apache/drill/jdbc/StatementTest.java

```diff
@@ -69,6 +78,17 @@ public static void tearDownStatement() throws SQLException {
     connection.close();
   }
 
+  @Before
```

Comment:
This will slow down all the tests in this class besides your tests.
Also, since connection string is `jdbc:drill:zk=local`, so separate embedded drill instance will be used for every test class and there is no need to lock other test classes.

Could you please move all your tests into a separate class and add lock there?

Reply:
Ok. I'll move only the `get|setMaxRows` related tests.